### ISSUE TITLE
Doesn't seem to protect at all

### ISSUE THREAD

**Issue by VanessaE** [stujones11/hazmat_suit#4]
Using Minetest 5.4.0 (exact release commit) and `minetest_game` from the same date.  My entire mods tree (every last mod I have) was updated earlier today, so I am certain that all the components listed below are completely up-to-date.  All of the mods on my PC are obtained from git where possible -- I do not use the Content Store.

I created a new world with `minetest_game`, enabled damage and disabled creative mode.  I enabled the following mods:

hazmat_suit
3d_armor
3d_armor_sfinv
pipeworks
technic
technic_worldgen
basic_materials

All of these are unaltered from their upstream content (I was NOT using anything from Dreambuilder, in case you may have considered that).

I started the world, gave myself a hazmat suit, and put it in the armor page to wear it.

I then gave myself a corium source and placed it.  Instant death.

The suit dropped on the ground when I died (but the rest of my holdings did not, as I have bones disabled).

I managed after about a dozen back-to-back deaths to remove the corium and recover the dropped suit.  Once I was able to get far enough away from the radiation to have some time between deaths, I put the suit back on.

I got close to the Chernobylite left behind from the corium destroying its surroundings.  It is deadly as well, though considerably slower to kill.

The suit shows that it's taken damage, but it didn't stop me from dying. :stuck_out_tongue:

**Comment by Conradish006** [comment]
Yes, the suit does absolutely nothing. Something else I noticed is where it's hard to jump up a node to keep walking only with the suit on. Might be a client issue IDK.

**Comment by mckaygerhard** [comment]
impossible.. cos since MT 5.3 this mod cannot be loaded due the translation unsupported ..

**Comment by mckaygerhard** [comment]
impossible.. cos since MT 5.3 this mod cannot be loaded due the translation unsupported ..  in my minetest sub nasa sevver it works.. but is using the older translation mod
